client/elements/sc-page-static.html

```diff
@@ -167,7 +170,7 @@ <h1 class="title-text">
       </div>
     </app-toolbar>
     <app-toolbar id="nav_toolbar" bottom-item>
-      <template is="dom-if" if="[[!shouldShowSecondToolbar]]">
+      <template is="dom-if" if="[[noToolbarSelected]]">
```

Comment:
I think `noToolbarSelected` would be better suited as a function `isToolbarSelected(shouldShowTipitakaToolbar, shouldShowSecondToolbar, shouldShowOrganizationalToolbar, shouldShowAcademicToolbar)`.